pkg/local_object_storage/engine/shards.go

```diff
@@ -60,6 +64,10 @@ func (e *StorageEngine) AddShard(opts ...shard.Option) (*shard.ID, error) {
 		return nil, fmt.Errorf("could not add %s shard: %w", sh.ID().String(), err)
 	}
 
+	if e.cfg.metrics != nil {
+		e.cfg.metrics.SetReadonly(sh.ID().String(), sh.GetMode() != mode.ReadWrite)
```

Comment:
Let's use `GetMode().ReadOnly()` -- we don't use `mode.Degraded` at all, but if we want, we shouldn't report `readonly`, another metric can be done. But again, `Degraded` is not used.

Reply:
Ok, we use this for alerting, let's postpone it.